olm_deploy/manifests/deployment.yaml

```diff
@@ -4,15 +4,18 @@ metadata:
   name: metering-ansible-operator-registry
   labels:
     registry.operator.metering-ansible: "true"
+    tenant: ""
```

Comment:
This could probably be changed to match the above label and have a `registry.*` prefix too.